### PR TITLE
Add search loading message

### LINKED ISSUE #222: There should be friendly 'searching' message
Sometimes searching can take a few second (slow network, many results). A message saying something like 'searching' would be very useful.

### PR DESCRIPTION
**Pull request by WPFerg**
closes #222

### CODE REVIEW

**Comment by bjedrzejewski** [comment]
Reviewing :)

File: src/sidebars/search/search.html [hunk 1]
```diff
@@ -18,7 +21,7 @@
                   </div>
               </div>
           </div>
-          <div class="no-results" ng-show="searchCtrl.noResults">
+          <div class="results-message no-results" ng-show="searchCtrl.noResults">
```
Comment (owennw) [inline src/sidebars/search/search.html]: Might be a bit late here, but is the `no-results` class still being used?

Comment (WPFerg) [inline src/sidebars/search/search.html]: No, but I wanted to keep it so there's at least some differentiation between the two messages.

Comment (owennw) [inline src/sidebars/search/search.html]: Ah I see: `loading-message` is also a description?

Comment (WPFerg) [inline src/sidebars/search/search.html]: Correct

**Pull request merged**